tests/acceptance/AdvancedSettingsAnonDataToggleCest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class AdvancedSettingsAnonDataToggleCest {
+  function toggleAnonDataSetting(\AcceptanceTester $I) {
+    $I->wantTo('Confirm anon data settings can be toggled');
+    $noAnonData = '[data-automation-id="analytics-no"]';
+    $yesAnonData = '[data-automation-id="analytics-yes"]';
+    $I->login();
+    $I->amOnMailPoetPage('Settings');
+    $I->click('[data-automation-id="settings-advanced-tab"]');
+    $I->waitForText('Bounce email address');
+    $I->selectOption($noAnonData, 'No');
+    $I->seeOptionIsSelected($noAnonData, 'No');
+    $I->selectOption($yesAnonData, 'Yes');
+    $I->seeOptionIsSelected($yesAnonData, 'Yes');
```

Comment:
I think one way to check if this was saved in the DB is to refresh the page.

Reply:
Hahah, smart and should have been obvious. Done. I saved between toggles, as well.